tests/lax_numpy_test.py

```diff
@@ -5379,6 +5379,53 @@ def testIssue2347(self):
     np_object_list = np.array(object_list)
     self.assertRaises(TypeError, jnp.array, np_object_list)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_shape={}".format(
+          jtu.format_shape_dtype_string(shape, dtype)),
+       "shape": shape, "dtype": dtype}
+      for shape in all_shapes
+      for dtype in complex_dtypes))
+  def testLogaddexpComplex(self, shape, dtype):
+    rng = jtu.rand_some_nan(self.rng())
+    args_maker = lambda: (rng(shape, dtype), rng(shape, dtype))
+
+    @jtu.ignore_warning(category=RuntimeWarning, message="invalid value.*")
+    def np_fun(x1, x2):
+      return np.log(np.exp(x1) + np.exp(x2))
+
+    @jtu.ignore_warning(category=RuntimeWarning, message="invalid value.*")
```

Comment:
I don't think this decorator should be required on the jnp function

Reply:
It's not for the jnp function, but `(y.imag + np.pi) % (2 * np.pi)` may raise the exception if `y.imag` contains `nan`.